database/migrations/2021_03_18_100000_add_phone_areas.php

```diff
@@ -0,0 +1,1646 @@
+                [
+                    'code' => 360,
+                    'state_abbreviation' => 'WA',
+                    'note' => 'W Washington State: Olympia, Bellingham (area circling 206, 253, and 425; split from 206; see overlay 564)'
+            ],
+                [
+                    'code' => 425,
+                    'state_abbreviation' => 'WA',
+                    'note' => 'Washington: North Tier - Everett, Bellevue (split from 206, see also 253; overlay 564)'
+            ],
+                [
+                    'code' => 509,
+                    'state_abbreviation' => 'WA',
+                    'note' => 'E and Central Washington state: Spokane, Yakima, Walla Walla, Ellensburg'
+            ],
+                [
+                    'code' => 564,
+                    'state_abbreviation' => 'WA',
+                    'note' => 'W Washington State: Olympia, Bellingham (overlaid on 360; see also 206, 253, 425; assigned but not in use)'
+            ],
+                [
+                    'code' => 262,
+                    'state_abbreviation' => 'WI',
+                    'note' => 'SE Wisconsin: counties of Kenosha, Ozaukee, Racine, Walworth, Washington, Waukesha (split from 414)'
+            ],
+                [
+                    'code' => 414,
+                    'state_abbreviation' => 'WI',
+                    'note' => 'SE Wisconsin: Milwaukee County (see splits 920, 262)'
+            ],
+                [
+                    'code' => 608,
+                    'state_abbreviation' => 'WI',
+                    'note' => 'SW Wisconsin: Madison'
+            ],
+                [
+                    'code' => 715,
+                    'state_abbreviation' => 'WI',
+                    'note' => 'N Wisconsin: Eau Claire, Wausau, Superior'
+            ],
+                [
+                    'code' => 920,
+                    'state_abbreviation' => 'WI',
+                    'note' => 'NE Wisconsin: Appleton, Green Bay, Sheboygan, Fond du Lac (from Beaver Dam NE to Oshkosh, Appleton, and Door County; part of what used to be 414)'
+            ],
+                [
+                    'code' => 304,
+                    'state_abbreviation' => 'WV',
+                    'note' => 'West Virginia (see overlay 681)'
+            ],
+                [
+                    'code' => 681,
+                    'state_abbreviation' => 'WV',
+                    'note' => 'West Virginia (overlaid on 304)'
+            ],
+                [
+                    'code' => 307,
+                    'state_abbreviation' => 'WY',
+                    'note' => 'Wyoming'
+            ],
+        ];
+
+        $now = Carbon::now()->format('Y-m-d H:i:s');
+        $phoneAreaRecords = collect($phoneAreaData)
+            ->map(function (array $record) use ($now) {
+                $stateId = State::fromAbbreviation($record['state_abbreviation'])->getId();
+                unset($record['state_abbreviation']);
```

Comment:
This is a slick implementation! :octocat:

Reply:
Thank you :woman-tipping-hand::skin-tone-4: